src/main/docker/gitlab-gitlabci.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.6"
+services:
+    gitlab:
+        build: gitlab
+        volumes:
+            - artemis-gitlab-data:/var/opt/gitlab
+            - artemis-gitlab-logs:/var/log/gitlab
+            - artemis-gitlab-config:/etc/gitlab
+        hostname: 'HOSTNAME'
```

Comment:
Why did you add this?

Reply:
Good point. I had a problem with it before, so I added it, but actually you don't need it.